joomla/joomla-cms
# [4.0] Mail Templates SMTP password

Updates the field to use the new lock option

Pull Request for #31917

To test
1. System -> Mail Templates
2. Options
3. Per Template Mail Settings -> Yes
4. Open any mail template and go to options tab
5. Mailer -> SMTP
6. SMTP Authentication -> Yes
7. Observe SMTP Password field

### Before
![image](https://user-images.githubusercontent.com/1296369/117637519-c50c2a00-b179-11eb-92dc-fc4b0192dd30.png)

### After

![image](https://user-images.githubusercontent.com/1296369/117637427-b1f95a00-b179-11eb-810e-93771ce20d97.png)
